lms/djangoapps/courseware/tests/__init__.py

```diff
@@ -1,141 +0,0 @@
-"""
```

Comment:
Reviewers: I moved this code to helpers.py as otherwise there are circular imports when accessing this package.